### ISSUE TITLE
Inaccessibility: Hierarchy of heading levels

### ISSUE THREAD

**Issue by shawnthompson** [a11yproject/a11yproject.com#231]
The heading levels on the index, resources and about pages are missing a level.
For example, on the index page you have the title, "The Accessibility Project", as the h1 followed by "Latest Posts" as an h2 but then your post titles are all h4. Therefore you're missing an h3 on that page.

AT device users who navigate via heading level might think there are not any levels after h2 and stop there without checking for the 4th level headings.

Also the footer headings start at h5 on every page but not every pages goes that deep in heading levels. My suggestion would be to create an invisible h1 (footer) to group your footer sections and starting them at h2.

**Comment by svinkle** [comment]
This issue has become "[stale](https://github.com/a11yproject/a11yproject.com/blob/gh-pages/CONTRIBUTING.md#stale-issues)" — @shawnthompson, feel free to open a [new issue](https://github.com/a11yproject/a11yproject.com/issues/new) if you'd like to continue.
